### DISCUSSION TITLE
How to turn off multi select?

### DISCUSSION THREAD

**Discussion by dkulakov** [TanStack/table#2126] [General]
Hi all!

Does useTable have some option which turns off multi-select for useRowSelect hook?
I need to have behavior when a user can select only one item in the table.

Attach gif demo

![useRowSelect](https://user-images.githubusercontent.com/6547279/78871596-9b517280-7a50-11ea-95ec-f9df8bcfd3c9.gif)

**Comment by gargroh** [comment]
Use can use direct methods on `instance` object https://github.com/tannerlinsley/react-table/blob/master/docs/api/useRowSelect.md#instance-properties

**Reply by dkulakov** [reply]
@gargroh thank you for your answer!

**Reply by dkulakov** [reply]
Is that what you meant?

![image](https://user-images.githubusercontent.com/6547279/78999798-0afc5600-7b54-11ea-8fb0-791bcf1aa9da.png)

**Reply by dkulakov** [reply]
https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/streaming-rows?file=/src/App.js

**Comment by tahv0** [comment]
@gargroh  your link is broken. Any chance to get correct link?

**Reply by dkulakov** [reply]
https://react-table.tanstack.com/docs/api/useRowSelect#instance-properties
